src/os_win.cc

```diff
@@ -33,6 +33,63 @@
 
 namespace toit {
 
+static wchar_t* malloced_wide_string(const char* string) {
```

Comment:
The names of these functions are kinda weird. They are sort of based on the type, because they are conversions. I think it would be more natural to have them named on what they do -- something like convert or allocate instead of converted or allocated.

`malloc_wide_string`
`new_wide_string`
`convert_to_wide_string`
`to_wide_string`

Reply:
Went with `to_wide_string` and `to_narrow_string`.
done.